xbmc/interfaces/legacy/RenderCapture.h

```diff
@@ -35,32 +34,36 @@ namespace XBMCAddon
 
     class RenderCapture : public AddonClass
     {
-      CRenderCapture* m_capture;
+      unsigned int m_captureId;
```

Comment:
@Memphiz if all goes right an addon should not care. It is set in the capture(), line 112. what error message do you get?